rest_test/debugging_tools/test.go

```diff
@@ -0,0 +1,363 @@
+////////////////////////////////////////////////////////////////////////////////
+////
+//// **IMPORTANT NOTE FOR CODE REVIEW**
+////
+//// This file is used for testing and debugging. Much of the code it contains
+//// is similar or the same to the `hooks.go` file.
+////
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Suggestion: name this file to clarify that it sets up the tests, rather than being purely a debugging tool. It'll help our future selves when we come back to update it and have forgotten how it all worked. Something like `setup/setup_test.go` ?

Reply:
🆗 👌